### DISCUSSION TITLE
The docker image qodana has expired.I want to check the Java code. Which image should I use？

### DISCUSSION THREAD

**Discussion by huo2huo** [JetBrains/Qodana#83] [Q&A] [answered]
The docker image qodana has expired.I want to check the Java code. Which image should I use？
![image](https://user-images.githubusercontent.com/101409437/164689519-fa401640-2ecd-49d1-9cbe-6223059951b1.png)

**Comment by brichbash** [comment, marked as answer]
HI!
You can pull a new image running `docker pull jetbrains/qodana:latest` or you can add the flag `--pull always` to your command that will always download a new image when it is available. By the way, it's better to use jetbrains/qodana-jvm-community or jetbrains/qodana-jvm for Java.

**Reply by huo2huo** [reply]
Thank you. jetbrains/qodana-jvm-community or jetbrains/qodana-jvm is really better.
